src/components/singleProductPage/singleProductDetails.scss

```diff
@@ -0,0 +1,369 @@
+@import "../common/variables";
+
+.singleProductContainer {
+  height: 100vh;
+}
+
+.productDetailsRow {
+  margin-bottom: 5%;
+}
+
+.productInfoContainer {
+  width: 100%;
+  height: 70vh;
+  display: flex;
+  margin: auto;
+}
+
+.productImgs {
+  .slick-slide > div {
+    width: 100%;
+  }
+
+  width: 50%;
+  height: 100%;
+  background-color: #ecebe4;
+  margin: 3%;
+  display: flex;
+
+  .mainImg {
+    width: 70%;
+    background-color: transparent;
+    margin: auto;
+  }
+
+  .smallImgs {
+    width: 400%;
```

Comment:
using width above 100% seems a bit suspicious to me, this margin top is suspicious too. perhaps you want a display-block here? if you are going in this route, then you should add comments about why this weird css is being sued

Reply:
so I made them 400% because I used slick here and each image is li which is a slick dot so there 100% is so tiny